Vagrantfile

```diff
@@ -3,19 +3,20 @@
 
 require 'yaml'
 
-ansible_vars = YAML.load_file("provisioning/vars.yml")
+common_ansible_vars = YAML.load_file("provisioning/vars.yml")
+vagrant_ansible_vars = YAML.load_file("provisioning/vagrant-vars.yml")
 
-app_name = ansible_vars["nodejs_app_name"]
+app_name = common_ansible_vars["nodejs_app_name"]
 
-app_directory = ansible_vars["nodejs_app_install_dir"]
+app_directory = vagrant_ansible_vars["nodejs_app_install_dir"]
```

Comment:
``vagrant_ansible_vars`` seems fine (maybe it could even be shortened to ``vagrant_vars``) but in my opinion the ``common_ansible_vars`` changes don't seem necessary.